src/sensors.py

```diff
@@ -315,3 +372,29 @@ def setup_sensors(instrument, katcp_server):
         katcp_server.add_sensor(sensor)
         instrument._sensors[sensor.name] = sensor
         ioloop.add_callback(_feng_pfb_okay, sensor, executor, _f)
+
+    # f-engine comms rx
+    fengops = corr2.fxcorrelator_fengops.FEngineOperations(instrument)
```

Comment:
By the way, there is a newer, sexier way to define sensors. Instead of Sensor(sensor_type=Sensor.TYPE, ...) you can do Sensor.type(...), quite a bit shorter :)